### PR TITLE
Update Indonesian Translation

### LINKED ISSUE #1582: All languages need to be updated for 0.13 - Call for translators
Another hard fork is approaching, and with it, a mayor release: 0.13
Since 0.12.3, the GUI changed a bit. Some strings were added and some were removed. #1572 refreshed all language files, so now **they can all be updated with the missing translations**, before 0.13 is out.

**This is a call for translators**, down here can be found the list of the available languages. If the language is thicked, somebody worked on it and the translation is already PRd.

**If you wish to contribute, please make sure your language is still labelled as 'needed' on the [Issue tracker of the Localization Workgroup on Taiga](https://taiga.getmonero.org/project/erciccione-monero-localization/taskboard/test)**. The task related to this issue is "Update strings (0.13)". If the task is 

+ **Needed**: nobody worked on it and still need a translator.
+ **Work in progress**: somebody is actively working on the language, but haven't opened a PR yet. 
+ **Ready for test**: the translation was PRd on this repository (the link to the PR can be found in the description of the task)

**If you start to work on a language, please write here or let me know** using one of the communication channel listed below, so that i can update the issue tracker and avoid double works.

- [x] **Arabic** - #1591 - *Merged*
- [ ] **Bulgarian** - New! Merged in #1637 , but won't be included in 0.13
- [x] **Catalan** - #1620 (not sure it will make it for 0.13)
- [x] **Chinese (Simplified)** - #1600 - *Merged*
- [x] **Chinese (Taiwanese)** - #1581 - *Merged*
- [ ] **Croatian**
- [x] **Czech** - #1618 - *Merged*
- [x] **Danish** - #1556 - *Merged*
- [x] **Dutch** - ~~#1594~~ replaced by #1599 - *Merged*
- [ ] **Esperanto**
- [x] **Finnish** - #1593 - *Merged*
- [x] **French** - #1586 - *Merged*
- [x] **German** - #1588 - *Merged*
- [x] **Hebrew** - #1603 - *Merged*
- [x] **Hungarian** - #1612 - *Merged*
- [ ] **Indonesian**
- [x] **Italian** - #1584 - *Merged*
- [x] **Japanese** - #1597 - *Merged*
- [ ] **Korean**
- [x] **Lithuanian** - #1608 - *Merged*
- [ ] **Pirate**
- [x] **Polish** - #1589 - *Merged*
- [x] **Portuguese (pt)** - #1596 - *Merged*
- [x] **Portuguese (br)** - #1590 - *Merged*
- [x] **Romanian** - #1607 
- [x] **Russian** - #1598 - *Merged*
- [x] **Serbian** - #1611 - *Merged*
- [x] **Slovak** - #1592 - *Merged*
- [x] **Slovenian** - #1613 - *Merged*
- [x] **Spanish** - #1601 - *Merged*
- [x] **Swedish** - #1609 - *Merged*
- [x] **Turkish** - #1585 
- [ ] **Ukrainian**

## How to translate?
Before starting the translation, we ask you to follow these few guidelines:

+ When possible, do not use gender specific terminology
+ If available, use the glossary for your language. This will help keep translations consistent over different works. Available glossaries are: [Italian](https://github.com/monero-ecosystem/monero-translations/blob/master/italian-terminology.md), [German](https://github.com/monero-ecosystem/monero-translations/blob/master/german-terminology.md), [Swedish](https://github.com/monero-ecosystem/monero-translations/blob/master/swedish-terminology.md),  [Spanish](https://github.com/monero-ecosystem/monero-translations/blob/master/spanish-terminology.md) and [French](https://github.com/monero-ecosystem/monero-translations/blob/master/french-terminology.md)
+ If a string contains number, just keep them as they are
+ Read the guide [Translation tips for Monero translators](https://github.com/monero-ecosystem/monero-translations/blob/master/translation-tips.md), Which cointains many useful tips and tools.

**A step-by-step guide on how to translate the GUI wallet** is available [here](https://taiga.getmonero.org/project/erciccione-monero-localization/wiki/translating-the-gui) (the guide is about how to add a new language, but the instructions are also valid for updating one), but, basically, what a translator need to do is:

1. Look for the correct language file in [monero-gui/translations](https://github.com/monero-project/monero-gui/tree/master/translations)
2. Find the strings labelled `<translation type="unfinished">`
3. Add the correct translation of the `<source>` string between the `<translation>` label (removing the `type="unfinished"`tag)
4. when done, commit and open a Pull Request with your changes. ***Please refer to this issue (#1582) when the PR is opened**.

## Support
If you need help/informations, or have dubts about the workflow, post on this thread or contact me on these channels:

+ *#monero-translations* - Live support chat based on Freenode/IRC, but relayed on matrix/riot and MatterMost
+ *reddit*- just pm /u/ErCiccione
+ *email* - translate[at]getmonero[dot]org
+ *Twitter* - DM @calciferciccio
+ *Wire* - @ErCiccione

### Ping them all!
Pinging contributors who helped with translations in past and members of the Monero Localization Workgroup:

@3b7ameed @edwardlow @ruzaq @cryptobench @falcongoat @Rafficer @lovvskillz @Keksoj @i3visio @lh1008 @cryptochangements34 @glv2 @assylias @milargos @jonahar @TasmaniaKrama @hrumag @kenshi84 @takuto-h @isaacdigs @ProkhorZ @ElectricSlacks @fridzema @einsteinsfool @szogun1987 @netrik182 @rmbb @ordtrogen @mandrill-pie @fero-sk @jernejml @xmronadaily @alexej996 @snaggen @rtonline @zhizhongzhiwai @Lafudoci @el00ruobuob @Leza89 @turossztrapacska @rpinola @monerorus @kasperaitis @Fajl @TheFuzzStone @IST34 @kerastinell @MalMen

### PR DESCRIPTION
**Pull request by geomars**
What this PR do?
- [x] Fix #1582 (translation request)
- [x] Replace PR #1678 after previously did `lupdate -pro monero-wallet-gui.pro -ts translations/monero-core_id.ts -no-obsolete`

### CODE REVIEW

File: translations/monero-core_id.ts [hunk 1]
```diff
@@ -1079,404 +1079,411 @@ The following information will be deleted
...
-        <translation type="unfinished"></translation>
+        <source>Or manually blackball/unblackball a single output:</source>
+        <translation>Atau secara manual mem-blackball/mengurungkan blackball suatu keluaran:</translation>
     </message>
     <message>
         <location filename="../pages/SharedRingDB.qml" line="215"/>
         <source>Paste output amount</source>
-        <translation type="unfinished"></translation>
+        <translation>Tempelkan jumlah keluaran</translation>
+    </message>
```
Comment (erciccione) [inline translations/monero-core_id.ts]: Double closing tag here. Remove this string please

File: translations/monero-core_id.ts [hunk 2]
```diff
@@ -2603,254 +2613,247 @@ Payment ID: </source>
...
 Amount: </source>
         <translation>Jumlah: </translation>
     </message>
     <message>
         <location filename="../main.qml" line="636"/>
-        <location filename="../main.qml" line="754"/>
+        <location filename="../main.qml" line="751"/>
         <source>
 Fee: </source>
         <translation>Biaya</translation>
```
Comment (erciccione) [inline translations/monero-core_id.ts]: Please add `:` + whitespace

File: translations/monero-core_id.ts (hunk 1 above)
Comment (geomars) [reply translations/monero-core_id.ts]: Done

**Comment by erciccione** [comment]
Great. Can you squash those commits now?
If you use git:

1. `git rebase -i HEAD~2`
2. Change the second `pick` to `squash`
3. force push to your remote branch using `git push -f`

If you don't know git, add me as collaborator to your repo and i can fix it for you.

**Comment by geomars** [comment]
> Great. Can you squash those commits now?
> If you use git:
> 
> 1. `git rebase -i HEAD~2`
> 2. Change the second `pick` to `squash`
> 3. force push to your remote branch using `git push -f`
> 
> If you don't know git, add me as collaborator to your repo and i can fix it for you.

I know git, but I prefer add you as collaborator in my fork repo.

**Comment by geomars** [comment]
> Great. Can you squash those commits now?
> If you use git:
> 
> 1. `git rebase -i HEAD~2`
> 2. Change the second `pick` to `squash`
> 3. force push to your remote branch using `git push -f`
> 
> If you don't know git, add me as collaborator to your repo and i can fix it for you.

I know git but prefer to add you as collaborator in my fork repo, in case any messed up

**Comment by Monero-Pootle** [comment]
@geomars please rebase

**Comment by erciccione** [comment]
I forgot to change user. @Monero-Pootle it's me with the pootle account

**Comment by geomars** [comment]
@Monero-Pootle, @erciccione did rebase and change some line post rebase

**Comment by erciccione** [comment]
Perfect, thanks. Cna you squash in just one commit now? THen it can be merged

**Pull request merged**